config/core/resources/broker.yaml

```diff
@@ -128,8 +128,6 @@ spec:
                 description: 'Broker is Addressable. It exposes the endpoint as an
                     URI to get events delivered into the Broker mesh.'
                 type: object
-                required:
-                  - url
```

Comment:
why was it required in first place ? :thinking:

Reply:
It's required in the happy case, not in all cases, see https://github.com/knative/eventing/issues/3854.